lib/cypress_on_rails/middleware.rb

```diff
@@ -54,9 +54,11 @@ def handle_command(req)
       logger.info "handle_command: #{body}"
       commands = Command.from_body(body, configuration)
       missing_command = commands.find {|command| !@file.exists?(command.file_path) }
+
       if missing_command.nil?
-        commands.each { |command| @command_executor.load(command.file_path, command.options) }
-        [201, {}, ['success']]
+        results = commands.map { |command| @command_executor.load(command.file_path, command.options) }
+
+        [201, {}, [results.flatten.to_json]]
```

Comment:
hmmm, putting the `to_json` inside the middleware means every command has to return something responds to `to_json`. This could be a Gotcha for existing commands or custom commands that people have written.

options:
 * leave it to command itself to decide what it wants to return
 * check if the result responds to_json

what do you think?